app/views/transferComplete.scala.html

```diff
@@ -8,8 +8,11 @@
             <h1 class="govuk-panel__title">
                 @Messages("transferComplete.title")
             </h1>
-            <div class="govuk-panel__body">
-                @Messages("transferComplete.body")
+            <div class="govuk-panel__body govuk-!-font-size-27">
+                <span>@Messages("transferComplete.body")</span><br/><br/>
```

Comment:
Is it good practice to use the `<br/>` HTML element to create line spaces? Not sure

Reply:
Thanks for reminding me. It's not good practice. The reason behind this was that using the paragraph would mean having to create CSS just for this which is something I wanted to avoid. But I am just working out a different solution.

Reply:
Worked out a better way and will change it :)